jupyterhub/orm.py

```diff
@@ -393,28 +414,57 @@ def __repr__(self):
 
     def new_api_token(self, token=None):
         """Create a new API token
-        
+
         If `token` is given, load that token.
         """
         return APIToken.new(token=token, user=self)
 
     @classmethod
     def find(cls, db, name):
         """Find a user by name.
-
         Returns None if not found.
         """
         return db.query(cls).filter(cls.name==name).first()
 
 
+class UserServer(Base):
+    """The UserServer table
+    Each user can have have more than one server,
```

Comment:
This is a bit unclear to me as to what the table is storing.

My understanding is:
```
A table storing the One-To-Many relationship between a user and servers.

Each user may have one or more servers.

A server can have only one (1) user. This condition is maintained by UniqueConstraint.
```